### DISCUSSION TITLE
Storage Progress Bar

### DISCUSSION THREAD

**Discussion by fadethegap** [supabase/supabase#6879] [Questions] [answered]
I would like to add a file upload progress bar to my app. Does anyone have example code or recommendations on how to interact with supabase to determine upload progress?

**Comment by GaryAustin1** [comment, marked as answer]
I believe this issue is the most info on that right now.  https://github.com/supabase/storage-api/issues/23

**Reply by fadethegap** [reply]
Thank you @GaryAustin1. I can monkey with the httpRequest on the client side.
